### DISCUSSION TITLE
A very nice project, Kubespider uses this as a download softwear.

### DISCUSSION THREAD

**Discussion by richardli1598** [yutto-dev/yutto#200] [General]
We use yutto to download video automatically： https://github.com/opennaslab/kubespider/blob/main/docs/zh/user_guide/yutto_download_provider/README.md

Can we add an adapter chapter in readme:

### Adapter:
* [Kubespider](https://github.com/opennaslab/kubespider): With yutto, downloading video from bilibili automatically.

**Comment by SigureMo** [comment]
Thank you for liking it.

> Can we add an adapter chapter in readme:

Of course. Feel free to open a pull request for this.
